docs/examples/game-of-chance.md

```diff
@@ -1,271 +1,150 @@
 ---
-sidebar_label: Game of chance
 sidebar_position: 9
 ---
 
 # Game of chance
 
 ## Introduction
 
-Anyone can easily create their own game application and run it on the Gear Network. To do this, Gear created an example of the Game-of-chance smart contract, which is available on [GitHub](https://github.com/gear-dapps/lottery). 
+:::note
 
-This article explains the programming interface, data structure, basic functions their purpose. It can be used as is or modified to suit your own scenarios.
+This article explains at a superficial level the purpose and logic of this smart contract. For a more detailed technical description, see its [documentation on the dApps documentation portal](https://dapps.gear.rs/game_of_chance) and [the source code section](#source-code).
 
-Gear also [provides](https://github.com/gear-tech/gear-js/tree/master/apps/game-of-chance) an example implementation of the [Game of chance's](https://lottery.gear-tech.io/) user interface to demonstrate its interaction with smart contracts in the Gear Network. In this example, whoever initializes the contract is considered the game owner. Only the owner has the right to start/finish the game. Players are added to the Game of chance themselves by sending a message with their bet to the contract. Then players monitor the state of the game. The winner is determined randomly.
```

Comment:
I suggest to keep all this staff - links to frontend implementation and video. It will be updated and available in the same links.